tests/auto-alex-v3.test.ts

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+import { Cl, ClarityType } from '@stacks/transactions';
+import { beforeEach, describe, expect, it } from 'vitest';
+import {
+    createClientMockSetup
+} from './clients/mock-client';
+import { tx } from '@hirosystems/clarinet-sdk';
+
+const { contracts, oracle, user2, bot, operator3, prepareALEX } = createClientMockSetup();
+
+describe('auto-alex-v3', () => {
+    it('auto-alex-v3 : ensure that it works', () => {
+
+        const wallet_1 = oracle;
+        const wallet_2 = bot;
+        const wallet_3 = user2;
+        const wallet_4 = operator3;
+        const dx = 1e8;
+        const end_cycle = 32;
+        const ACTIVATION_BLOCK = 20;
+
+        let response = prepareALEX();
+        expect(response.result).toHaveClarityType(ClarityType.ResponseOk);
```

Comment:
check could be part of prepareAlex

Reply:
https://github.com/lisalab-io/liquid-stacking/pull/56/commits/20107362b60f7890c0538305e7089746f26b6ac3